### PR TITLE
Added SharmaRajan json file in Submission folder

### PR DESCRIPTION
**Pull request by SharmaRajan**

### CODE REVIEW

File: Submissions/SharmaRajan.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Rajan Kumar Sharma",
+  "img": "[YOUR_IMG_URL]",
```
Comment (sophiabrandt) [inline Submissions/SharmaRajan.json:3]: Please remove placeholder.

File: Submissions/SharmaRajan.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "email": "mr.sharmajeerajan@gmail.com",
+  "links": {
+    "website": "[YOUR_WEBSITE_URL]",
```
Comment (sophiabrandt) [inline Submissions/SharmaRajan.json:6]: Same here.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Comment by sophiabrandt** [comment]
@SharmaRajan can you please fix the above issues, so that we can merge your pull request?

**Comment by SharmaRajan** [comment]
Removed the placeholder @sophiabrandt

**Pull request merged**